assets/src/components/map/controls/LayersControl.tsx

```diff
@@ -0,0 +1,104 @@
+import Leaflet from "leaflet"
+import React, { useEffect, useState } from "react"
+import ReactDOM from "react-dom"
+import { TileLayer, useMap, useMapEvents } from "react-leaflet"
+import { joinClasses } from "../../../helpers/dom"
+import { TileType, tilesetUrlForType } from "../../../tilesetUrls"
+import { MapLayersIcon } from "../../../helpers/icon"
+
+export const LayersControl = ({
+  tileType,
+  setTileType,
+}: {
+  tileType: TileType
+  setTileType: (tileType: TileType) => void
+}): JSX.Element | null => {
+  const map = useMap()
+  const portalParent = map
+    .getContainer()
+    // TODO: handle two maps?
+    .querySelector(".leaflet-control-container .leaflet-top.leaflet-right")
```

Comment:
**thought:** I played around and logged the output of `map.getContainer()` to console and it's actually the `<div>` that has the classes `c-vehicle-map leaflet-container`, so I think this would be fine if we had multiple maps because they would be under different `c-vehicle-map` components.